src/lib/round-progress/round-progress.config.ts

```diff
@@ -12,6 +12,7 @@ const DEFAULTS = {
   clockwise: true,
   semicircle: false,
   rounded: false,
+  innerCircleFill: 'none',
```

Comment:
Also this should probably default to `transparent` since `none` isn't a valid color.